static/js/main.js

```diff
@@ -0,0 +1,4 @@
+// Fix ipfs.io/blog/* links loaded via DNSLink: https://github.com/ipfs/blog/issues/360
+if (window.location.hostname === 'ipfs.io' && window.location.pathname.startsWith('/blog')) {
```

Comment:
This will error on IE as it knows not of the `startsWith`. :trollface:

Reply:
fine c1fb4423095d29d629c77f894f9ad571ada14849 :triumph: